### PR TITLE
[PL] [00000] Adding varnish 2.1.5 formula

### PR DESCRIPTION
**Pull request by pasha**
Current production version as of 01/08/2016 v2.1.5

### CODE REVIEW

File: varnish2.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
+require 'formula'
+
+class Varnish2 < Formula
+  url 'http://repo.varnish-cache.org/source/varnish-2.1.5.tar.gz'
+  head 'http://repo.varnish-cache.org/source/varnish-3.0.0-beta1.tar.gz'
```
Comment (colinrymer) [inline varnish2.rb]: We should probably remove this, since we don't want folks to accidentally install v3.

Comment (pasha) [inline varnish2.rb]: good point, removing

**Comment by colinrymer** [comment]
LGTM, I say squash and merge. :+1:

**Comment by lfender6445** [comment]
how do i install this? tried:

```
brew tap rentpath/homebrew;
brew install rentpath/homebrew/varnish; brew install varnish2; brew install rentpath/homebrew/varnish2;
```

**Comment by lfender6445** [comment]
nvm, got it working with `brew install varnish2.rb` i guess tap just clones the repo from master rather than local source.

**Comment by pasha** [comment]
i used `brew install rentpath/homebrew/varnish2` i'll add it to readme

**Pull request merged**